cassandraprotocol/frame/rawframe.go

```diff
@@ -0,0 +1,38 @@
+package frame
+
+import (
+	"fmt"
+	"github.com/datastax/go-cassandra-native-protocol/cassandraprotocol"
+)
+
+// A low-level representation of a frame, where the body is not decoded.
+type RawFrame struct {
+	Header *RawHeader
+	Body   []byte
+}
+
+// A low-level representation of a frame header, as it is parsed from an encoded frame.
+type RawHeader struct {
+	IsResponse bool
+	Version    cassandraprotocol.ProtocolVersion
+	Flags      cassandraprotocol.HeaderFlag
+	StreamId   int16
+	OpCode     cassandraprotocol.OpCode
+	BodyLength int32
+}
+
```

Comment:
I suggest that to make things more self-explanatory we create a struct `RawBody struct { ... }` to hold the raw body, wdyt?

Reply:
you mean this?
```type RawBody struct {
    Body []byte
}```

Why not just change the names of the fields in the `RawFrame` struct if that's the concern?
```
type RawFrame struct {
    RawHeader *RawHeader
    RawBody    []byte
}
```

Reply:
Ok that works with me too.

Reply:
done